plugin-packs/postcss-preset-env/.tape.mjs

```diff
@@ -271,44 +280,13 @@ postcssTape(plugin)({
 			}
 		}
 	},
-	'insert:after:match-result:feature-is-run': {
-		message: 'supports { insertBefore } usage when looking for a result and the attached feature is skipped',
-		options: {
-			stage: 0,
-			browsers: [
-				'safari >= 15',
-			],
-			insertAfter: {
-				'lab-function': [
-					orderDetectionPlugin('after', (decl) => {
-						return decl.prop === 'color' && decl.value !== 'changed-this-declaration';
-					})
-				]
-			}
-		}
-	},
-	'insert:after:match-result:feature-is-skipped': {
-		message: 'supports { insertBefore } usage when looking for a result and the attached feature is skipped',
-		options: {
-			stage: 0,
-			browsers: [
-				'safari >= 15',
-			],
-			insertAfter: {
-				'lab-function': [
-					orderDetectionPlugin('after', (decl) => {
-						return decl.prop === 'color' && decl.value !== 'changed-this-declaration';
-					})
-				]
-			}
-		}
-	},
 	'insert:after:match-source': {
 		message: 'supports { insertAfter } usage when looking for source',
 		options: {
 			stage: 0,
 			features: {
-				'lab-function': true
+				'lab-function': true,
+				'color-function': false,
```

Comment:
`color-function` transforms `color(display-p3 ...)`.
These tests look for the results of `lab-function` transforms, so displaying `color-function` for now in these cases.

Maybe these need to be rewritten to use a more basic plugin?

Reply:
I thought it was related to this and also thought about using something else such as `rebeccapurple` or something that's less prone to change.

Reply:
Thanks for taking care of this!